pbsmetrics/go_metrics.go

```diff
@@ -526,6 +537,10 @@ func (me *Metrics) RecordPrebidCacheRequestTime(success bool, length time.Durati
 	}
 }
 
+func (me *Metrics) RecordRequestQueueTime(success bool, requestType RequestType, length time.Duration) {
+	me.RequestsQueueTimer[requestType][success].Update(length)
```

Comment:
This will produce a null reference exception if the requestType is not video. Not a problem, but please add a check here to ignore all non-video requests to be sure.

Reply:
Added `requestType` check